src/schema/metadata/StationName.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: StationName
+description: |
+  Institution defined name of the machine that produced the composite
+  instances.
```

Comment:
should we change all "composite instances" to "measurement"? see also https://github.com/bids-standard/bids-specification/pull/813

Reply:
So confused... I totally thought I got them all in 6ebde13475dfadc3b76c075d3d69a82de734fdb9. I'll hunt down the rest of the "composite instances" and fix them.

Reply:
Fixed in 90c00d2.